gazelle/python/testdata/directive_python_test_file_pattern/README.md

```diff
@@ -0,0 +1,18 @@
+# Directive: `python_test_file_pattern`
```

Comment:
Naming. I also considered these other options:

+ `python_test_file_patterns`  (plural)
+ `python_test_file_glob`
+ `python_test_file_globs`

LMK if you'd prefer one of those or an entirely new one.

Reply:
The current choice is good.